ocaml/util/jbuild

```diff
@@ -1,6 +1,17 @@
+(* -*- tuareg -*- *)
+#require "unix"
+
+let coverage_rewriter =
+  let is_coverage = try Unix.getenv "BISECT_ENABLE" = "YES" with Not_found -> false in
+  if is_coverage then
+    "(preprocess (pps (bisect_ppx -conditional)))"
+  else
+    ""
+
+let () = Printf.ksprintf Jbuild_plugin.V1.send {|
 (rule
  ((targets (build_info.ml))
-  (action  (with-stdout-to ${@} (run "date" "+let date=\"%Y-%m-%d\"")))))
+  (action  (with-stdout-to ${@} (run "date" "+let date=\"%%Y-%%m-%%d\"")))))
```

Comment:
Why you need to double the `%`?

Reply:
this is part of the `Printf` above, we need the `%` to be interpreted by the `date` command, not by Printf, hence we need to escape it.

Reply:
Oh right, thanks